src/components/QRCode.tsx

```diff
@@ -10,13 +10,18 @@ const ResponsiveSvgWrapper = styled.div`
   }
 `
 
-interface Props {
+interface QRCodeProps {
   value: string | Uint8Array
+  size?: number
+  bgColor?: string
+  fgColor?: string
+  level?: 'L' | 'M' | 'Q' | 'H'
+  renderAs?: 'svg' | 'canvas'
 }
 
-const QRCode = ({ value }: Props) => (
+const QRCode = ({ level = 'H', renderAs = 'svg', value }: QRCodeProps) => (
```

Comment:
given that we know we're printing this and we know exactly how big it should be on paper, I wonder if we're better off rendering as a PNG given how SVG occasionally causes us trouble when printing.

Reply:
this was the printing issues with the Brother HL2300 and the SVG of the state seals. Not a blocker, just trying to play it defensive here since SVGs have been unfriendly to us in the past.

Reply:
That was 100% due to a horrible export to SVG by some wack tool. 

SVG will not be an issue for these QR codes.